module/zfs/spa_stats.c

```diff
@@ -728,4 +928,10 @@ MODULE_PARM_DESC(zfs_read_history_hits, "Include cache hits in read history");
 
 module_param(zfs_txg_history, int, 0644);
 MODULE_PARM_DESC(zfs_txg_history, "Historic statistics for the last N txgs");
+
+/* CSTYLED */
+module_param(zfs_multihost_history, int, 0644);
+MODULE_PARM_DESC(zfs_multihost_history,
+	"Historic statistics for last N multihost writes");
```

Comment:
We should say historical. Historic means "famous or important in history" :smile: